### ISSUE TITLE
Failed Promise in play()

### ISSUE THREAD

**Issue by dev-steverob** [quasarframework/app-extension-qmediaplayer#63]
https://github.com/quasarframework/app-extension-qmediaplayer/blob/a795b1e1030cc58398dde1cfcd209333ab65526e/src/component/QMediaPlayer.js#L494-L509

Sorry to be a pain! if the promise fails then it will still set the playing state to true as you change the state on line 496 and then don't change it back. You need to add a `.catch((e) => {})` to the `play()` promise eg:

```
this.$media.play().then(() => { 
  this.state.showBigPlayButton = false 
  this.state.playing = true 
  this.__mouseLeaveVideo() 
}).catch((err) => {
  console.error(err)
  this.state.showBigPlayButton = true 
  this.state.playing = false
})
```

**Comment by hawkeye64** [comment]
Crud! Thought I had removed that. Sorry.

**Comment by hawkeye64** [comment]
Fixed in v1.0.9
